src/fileManagement.js

```diff
@@ -253,7 +253,7 @@ class Files {
    * @returns {Promise.<status>}  boolean: whether the operation was successful
    * @returns {Promise.<error>}   string: error message, if any.
    */
-  move (source, target) {
+  move (source, target, overWrite = false) {
```

Comment:
please don't use camelcase - `overwrite` is one word, all lower case :-)

Reply:
ah sorry was copy pasta :D thx

Reply:
done